### PR TITLE
Updates ROADMAP with the 'Harder, Better, Faster, Stronger' release

### PR DESCRIPTION
**Pull request by tima**

### CODE REVIEW

File: ROADMAP.md [hunk 1]
```diff
@@ -1,6 +1,14 @@
 # Roadmap
 
-## Release 3.0 (Completed 30 June 2018)
+## Release 3.1 "Harder, Better, Faster, Stronger" (Target Release Date: 30 September 2018)
+
+- Static analysis of roles during import/upload using ansible-lint and additional Galaxy best practice rules
+- Roles quality scoring improved the search result ranking and presentation to make it even easier to find the best content
```
Comment (chouseknecht) [inline ROADMAP.md]: Poorly worded sentence.

File: ROADMAP.md [hunk 2]
```diff
@@ -1,6 +1,14 @@
...
+- Introduce versioned content
```
Comment (chouseknecht) [inline ROADMAP.md]: What does this mean, exactly?

File: ROADMAP.md [hunk 3]
```diff
@@ -1,6 +1,14 @@
...
+- Begin migration from a Github-specific pull model to an agnostic push model where contributors publish content collections that hosted on the public Galaxy server
```
Comment (chouseknecht) [inline ROADMAP.md]: Poorly worded.

File: ROADMAP.md (hunk 2 above)
Comment (tima) [reply ROADMAP.md]: It means galaxy content can have version numbers that will be enforced and managed.

**Pull request merged**